config/install/field.storage.localgov_alert_banner.field_localgov_type_of_alert.yml

```diff
@@ -0,0 +1,32 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - localgov_alert_banner
+    - options
+id: localgov_alert_banner.field_localgov_type_of_alert
```

Comment:
And so finally the storage, https://github.com/localgovdrupal/localgov_services/blob/b8d65a1a361a05ec368f3156f7d818c12bc93fff/modules/localgov_services_navigation/config/install/field.storage.node.localgov_services_parent.yml#L7

As for naming _these_ fields. I actually think you're free to call them whatever you want. Because the entity type itself is `localgov_`. I'd drop the `field_` bit just because it's not necessary, but it's not strictly required to include the `localgov_` bit.

Reply:
As per slack conversation, I've unprefixed the fields.
These match up with the base entity definations, and as user fields will typically be `field_` and modules their own prefix its safe to assume it won't conflict here. If users are changing or removing their own sites field prefixes, we can assume they understand the risks.